code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -717,6 +717,17 @@
 			limb.overlays += limb_em_block
 		return
 
+	//HUSK SHIIIIT
```

Comment:
Useless comments, remove it

Reply:
I very much like having comments breaking apart chunks of bodypart render code, it makes it significantly more readable.

Reply:
I disagree but don't care enough to block the PR on it